forem/forem
# Restore static error pages

## What type of PR is this? (check all applicable)


## Description

https://github.com/forem/forem/pull/11744 was not a great idea, even though it made sense, it defeated the purpose of serving error pages from the edge which lead to a huge spike in error messages server side.

What prompted was removing the "file an error on GitHub" notice as suggested by https://github.com/forem/forem/issues/11718 

The conditional is not worth the hit on the server, if we decide to bring it back we can find a way to do it with edge workers or something similar.

This reverts (in order): 

- https://github.com/forem/forem/pull/11806
- https://github.com/forem/forem/pull/11744

and bumps the service worker cache that was bumped to `1.3` in https://github.com/forem/forem/pull/11806 to `1.4` so we start with a clean slate.

## Related Tickets & Documents

- https://github.com/forem/forem/issues/11718
- https://github.com/forem/forem/pull/11806
- https://github.com/forem/forem/pull/11744

## QA Instructions, Screenshots, Recordings

1. set `config.consider_all_requests_local` to `false` in the file `config/environments/development.rb` file
1. load the web server
1. navigate to either http://localhost:3000/404, http://localhost:3000/422, http://localhost:3000/500, http://localhost:3000/503 to verify they work and are indeed the old pages, same as https://dev.to/404, https://dev.to/422, https://dev.to/500 and https://dev.to/503

## Added tests?
